edge/pkg/edged/dockertools/docker_manager.go

```diff
@@ -73,7 +73,7 @@ const (
 )
 
 //DockerDefaultAddress is default address for docker
-const DockerDefaultAddress = "unix:///var/run/docker.sock"
+var DockerDefaultAddress = "unix:///var/run/docker.sock"
```

Comment:
Any reason to change const to var? For docker **default** address, I think it's OK to take it as const.

Reply:
OK. I will define new variable for the DockerAddress